cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/management-cluster-settings-step/ManagementClusterSettings.test.tsx

```diff
@@ -0,0 +1,54 @@
+// React imports
+import React from 'react';
+
+// Library imports
+import { render, waitFor, screen, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ManagementClusterSettings from './ManagementClusterSettings';
+
+// App imports
+
+describe('ManagementClusterSettings', () => {
+    it('should render', async () => {
+        const view = render(<ManagementClusterSettings />);
+        expect(view).toBeDefined();
+    });
+
+    it('show a message if there is only one option for the control plane node profile', () => {
```

Comment:
nit: shouldn't this title be more generic: 'show a message if one is provided'?

Reply:
fixed.